dblp-aliases.csv

```diff
@@ -21833,6 +21833,7 @@ Frédérique Chaussumier,Frédérique Silber-Chaussumier
 Frédérique Mélanie-Bécquet,Frédérique Mélanie
 Frédérique Rémy,Frederique Remy
 Frédérique Sannier,Fred Sannier
+Frederic Sala,Fred Sala 0001
```

Comment:
Do not change this file - it is automatically generated.

Reply:
Ohh, I was following this instruction:
> If an alias is not already present in dblp-aliases.csv, add it.

Reply:
Ah, that is outdated. Thanks - I have fixed it.